### DISCUSSION TITLE
Failed config on extra.css or bug?

### DISCUSSION THREAD

**Discussion by MacGyver-NRW** [squidfunk/mkdocs-material#4867] [Q&A] [answered]
Hi Folks!
i did try out to color an icon like described on: https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#with-colors

It's a simple code in extra.css:
```
.red {
    color: crimson;
}
```

Of corse, i added the extra css in `mkdocs.yaml` as like this:

```
extra_css:
  - stylesheets/extra.css

```

Fancy thing - I'm wondering... if i start `mkdocs serve` and also `mkdocs serve --watch-theme` - there is no compiling of my extra.css.

But: If i start the build process with `mkdocs build -c` it will create a /stylesheets/extra.css with a working css.
You can see the **working** example under my temporary url: https://doc.clanfamily.ddnss.de/contribute/activities/#top-mitwirkende
I can't provide the "served" version.

Why dosn't this work in "serve Mode" ? Is there anything wrong with my exception? Or is this a possible bug?

Thanks for help ;)
Mirco

**Comment by squidfunk** [comment]
I'm not sure where the error is. Try switching to another theme and see if the error persists. If it does, it's not related to Material for MkDocs. If the issue doesn't persist, please attach a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/) here – we need something we can run to help you. Additionally, please read our [bug reporting guide](https://squidfunk.github.io/mkdocs-material/bug-report/).

**Comment by MacGyver-NRW** [comment, marked as answer]
@squidfunk i've got a clou... it seems to be a cache problem.
I just start up developing a few days later again - and my problem was solved.
Now i changed an other CSS Value - same ;) It won't update the cached css files.

I start my development by "mkdocs serve --watch-theme". Does it not compile changes on extra css?

**Reply by squidfunk** [reply]
MkDocs does not compile CSS, it only compiles Markdown sources. It sounds like a caching problem, where your browser will keep the old file, as the server sets the caching `max-age` header too far into the future.

Try doing a full reload, if that helps, it's definitely caching related.
